build-system/compile/bundles.config.extensions.json

```diff
@@ -630,9 +630,7 @@
     "version": "1.0",
     "latestVersion": "0.1",
     "options": {
-      "hasCss": true,
-      "npm": true,
-      "wrapper": "bento"
```

Comment:
I think this you're running into the same problem that I noticed [here](https://github.com/ampproject/amphtml/pull/36193#issuecomment-930559336), where any time there's an `export` in the root of `amp-foo.js` file, the compiler outputs a bundle with `export` declarations. Then wrapping that output in a function results in a compilation error because you can't have non-top-level exports.

Reply:
This is where I got stuck. Ideally we could configure the compiler to drop exports or extract only the `AMP.registerElement(...)` snippet and its dependenies, but not sure how to do that. I think @alanorozco had ideas here

Reply:
We're currently building `bento-*.js` files from the source on `amp-*.js`, but that's incorrect. [I'm addressing this on a different PR](https://github.com/ampproject/amphtml/pull/36174), and this might require a similar approach.

Bento Web Components should use the `BaseElement` implementation from `base-element.js` rather than `Amp*` from `amp-*.js`. 

This would allow us to ignore everything inside the `AMP.extension()` call, and avoid the wrapper and the `AMP.registerElement` call altogether. We also would prevent the nested `export` problem.

Then, a source `web-component.js` could either be added or generated. (The other PR [generates a similar `bento-timeago.js`](https://github.com/ampproject/amphtml/pull/36174/files#diff-586cddcb20b3d24635a884ba630a84119d063df527b9397d9aded7d16e2fb481R769-R779))

```js
// extensions/amp-timeago/1.0/build/web-component.js
import {BaseElement} from '../base-element';
import {CSS} from '../../../../build/bento-timeago-1.0.css';

export default function defineElement() {
  const style = document.createElement('style');
  style.textContent = CSS;
  document.head.appendChild(style);
  customElements.define('bento-timeago', BaseElement);
}
```